src/main/java/com/develcode/develfood/model/User.java

```diff
@@ -0,0 +1,83 @@
+package com.develcode.develfood.model;
+
+import com.develcode.develfood.model.enums.Role;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.Date;
+
+@Entity
+@Getter
+@Table(name = "users")
+@NoArgsConstructor
+public class User implements UserDetails {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    private String email;
+    
+    private String password;
+
+    private Date loginDate;
+
+    @Setter
+    private Role role;
+
+    @Setter
+    private boolean active;
```

Comment:
Rever necessidade desse campo

Reply:
Implementei esse boolean para poder efetuar a exclusão lógica das entidades, evitando a necessidade de executar delete na tabela.

Porém, se não for cabível agora, posso remover.